themes/src/main/themes/VAADIN/themes/valo/fonts/_fonts.scss

```diff
@@ -4,20 +4,27 @@
 @import "roboto/roboto";
 @import "lato/lato";
 @import "lora/lora";
+@import "../../../VAADIN/addons/vaadin-icons/vaadin-icons.scss";
 
-$v-font-awesome: true !default;
+$v-icons: true !default;
+$v-font-awesome: false !default;
```

Comment:
It would be good to document this variable, and note that it’s a legacy variable, and we encourage the use of the FontAwesome add-on from the directory.

For example:

```scss
/// Set the value of this variable to `true` if you want to use the Font Awesome icons from 
/// the Vaadin Framework 7 compatibility package. Note, that this is a legacy package, 
/// which will not be updated in the future. 
/// We encourage you to use the FontAwesomeLabel add-on from the Directory instead: 
/// https://vaadin.com/directory#!addon/fontawesomelabel
$v-font-awesome: false !default;
```

The other `$v-icons` variable would be good to document as well.

Are we planning on updating the currently deployed [Valo theme docs](https://vaadin.com/api/valo), btw?